hack/tools/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/golangci/golangci-lint v1.64.5
 	github.com/jcchavezs/porto v0.6.0
-	github.com/vektra/mockery/v2 v2.40.3
+	github.com/vektra/mockery/v2 v2.53.2
```

Comment:
This seems to be out of scope of this PR. Please, rebase PR branch on top of the latest master, thanks!